Python/Product/PythonTools/PythonTools/Navigation/EditFilter.cs

```diff
@@ -633,12 +633,20 @@ class NodeEnumerator<T> : IVsEnumNavInfoNodes where T : IVsNavInfoNode {
             if (pguidCmdGroup == VSConstants.GUID_VSStandardCommandSet97) {
                 switch ((VSConstants.VSStd97CmdID)nCmdID) {
                     case VSConstants.VSStd97CmdID.Paste:
-                        string updated = RemoveReplPrompts(_pyService, Clipboard.GetText(), _textView.Options.GetNewLineCharacter());
+                        string updated = RemoveReplPrompts(
```

Comment:
Better would be detecting what the editor is doing here.  There's a "VisualStudioEditorOperationsLineCutCopyClipboardTag" tag which is being added on the copy.  We can detect it and just let the editor handle the paste if it's present.